ams/server/src/main/java/com/netease/arctic/server/catalog/ExternalCatalog.java

```diff
@@ -15,19 +16,28 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.Callable;
+import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
 public class ExternalCatalog extends ServerCatalog {
 
   UnifiedCatalog unifiedCatalog;
   TableMetaStore tableMetaStore;
+  private Pattern tableFilterPattern;
 
   protected ExternalCatalog(CatalogMeta metadata) {
     super(metadata);
     this.tableMetaStore = CatalogUtil.buildMetaStore(metadata);
     this.unifiedCatalog =
         this.tableMetaStore.doAs(
             () -> new CommonUnifiedCatalog(this::getMetadata, Maps.newHashMap()));
+    if (metadata.getCatalogProperties().containsKey(CatalogMetaProperties.KEY_TABLE_FILTER)) {
```

Comment:
Can get CatalogMetaProperties.KEY_TABLE_FILTER directly, and then determine if tableFilter is null.

Reply:
Fixed.